### PR TITLE
Review of chapter chapter_post_and_database.asciidoc

### PR DESCRIPTION
**Pull request by Enforcer**
Another round of review from me.

Out of curiosity (I'm not a native English speaker) - what's with using "her" pronoun when gender is unknown? I only know about using they/them in such a situation.
I'm talking about a sentence that ends the first paragraph:
> so that we can save it somehow and display it back to her later

# General feedback
This chapter is quite long and reviewing it was a bit exhausting. Is there an option perhaps to split it somehow? Or put some summary in the middle to let a reader catch a breath?

### CODE REVIEW

**Comment by hjwp** [comment]
hey!  some people like to alternate between "he" and "her" pronouns when describing generic-example-people, because "they" feels awkward and impersonal, and the default we all tend to slip into otherwise (especially in a male-dominated world like programming) is to just use "he" everywhere...  so i try and keep the ratio about 50/50.

File: chapter_post_and_database.asciidoc [hunk 1]
```diff
@@ -1889,4 +1908,7 @@ The scratchpad to-do list::
 place to write down things that occur to us as we're coding, so that
     we can finish up what we're doing and come back to them later.
 
+// SEBASTIAN: (idea) alternative to maintaining a scratchpad could be to write empty unit tests without implementation.
+//    Such "tests prototypes" could be skipped initially until we work on them.
```
Comment (hjwp) [inline chapter_post_and_database.asciidoc:1912]: i've always like this idea in theory, but i've never found myself actually bothering to do it in practice, or not more than once or twice -- have you ever got into the habit?

**Comment by Enforcer** [comment]
> hey! some people like to alternate between "he" and "her" pronouns when describing generic-example-people, because "they" feels awkward and impersonal, and the default we all tend to slip into otherwise (especially in a male-dominated world like programming) is to just use "he" everywhere... so i try and keep the ratio about 50/50.

Oh, now I understand - thanks for taking the time to respond to that!

**Comment by ritaf-ORM** [comment]
The use of pronouns in examples have shifted over the past few years. Nowadays, it's part of our house style to use the generic pronouns they/their when needed, not he, she, or he/she.

**Pull request merged**